SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -84,7 +84,12 @@ lemma one_shiftLeft_mul_eq_shiftLeft {A B : BitVec w} :
   simp only [bv_toNat, Nat.mod_mul_mod, one_mul]
   rw [Nat.mul_comm]
 
+@[simp]
 def ofInt_zero_eq (w : Nat) : BitVec.ofInt w 0 = 0#w := rfl
+
+@[simp]
+def ofInt_one_eq (w : Nat) : BitVec.ofInt w 1 = 1#w := rfl
```

Comment:
```suggestion
@[simp]
lemma ofInt_one_eq (w : Nat) : BitVec.ofInt w 1 = 1#w := rfl
```
It should be `lemma` or `theorem`, but `def` is definitely wrong.

This should probably be a generic lemma about `OfNat.ofNat`, e.g., 
```lean
@[simp] lemma ofInt_ofNat (w n : Nat) : BitVec.ofInt w (OfNat.ofNat n) = n#w := by sorry
```

Reply:
I dropped this change.